connect-inject/envoy_sidecar.go

```diff
@@ -52,11 +52,27 @@ func (h *Handler) envoySidecar(pod *corev1.Pod) (corev1.Container, error) {
 			"--max-obj-name-len", "256",
 			"--config-path", "/consul/connect-inject/envoy-bootstrap.yaml",
 		},
-	}, nil
+	}
+	if h.ConsulCACert != "" {
+		caCertEnvVar := corev1.EnvVar{
```

Comment:
I don't personally see the benefit of these extra vars, but personal preference

Reply:
This is needed so that `preStop` command will succeed.